styles/items.css

```diff
@@ -9,49 +9,48 @@
 }
 
 #itemsDiv {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
   margin-bottom: 20rem; /* spacing so items dont get lost under sticky footer*/
 }
 
 .item {
   display: grid;
-  grid-template-columns: min-content 2fr 1fr;
+  grid-template-columns: min-content 1fr min-content;
   border-bottom: 1px solid #cdcccb;
-  background: var(--item-bg-color);
-  margin-bottom: 0.5rem;
+  background: white;
   font-size: 20px;
   width: 85%;
   margin: 0 auto;
+  border-radius: 8px;
 }
 
 .checkbox {
   height: 100%;
   width: 2rem;
-  margin-left: 0.5rem;
+  margin-left: 1rem;
 }
 
-/* container for an item in the rendered shopping list */
 .itemInfo {
-  display: grid;
-  grid-template-areas:
-    'name name'
-    'quantity price';
-  grid-template-columns: 1fr 1fr;
-  grid-template-rows: 1fr min-content;
-  gap: 0.7rem;
-  padding-left: 1rem;
   color: var(--text-color);
-  padding-top: 0.5rem;
-  padding-bottom: 0.2rem;
+  padding: 1rem;
 }
 
 .itemInfoHeading {
-  font-size: 14px;
+  font-size: 16px;
   font-style: italic;
   letter-spacing: 0.05rem;
 }
 
-.storeSection {
+.itemAmountSection {
   text-align: right;
+  font-size: 18px;
+  letter-spacing: 0.05rem;
+}
```

Comment:
This is causing some spacing issues, where a two word list item runs to 2 lines when it really doesn't seem like it needs to (see "Bread Flour" in the PR screenshot)

If the amount was absolutely positioned it wouldn't impact the space provided to the name of the list item.

Reply:
<img width="384" alt="image" src="https://github.com/user-attachments/assets/ead9bcce-0da8-43c5-9a6c-84b61af045c9">

This has been fixed 👍